README.md

```diff
@@ -1625,12 +1628,15 @@ It keeps track of the IDs of created resources so that Terraform knows what it i
 <details>
 <summary>Explain what the following commands do:
 
-  * <code>terraform init</code> initialises and downloads the provider's binary as described in the .tf files
-  * <code>terraform plan</code>	is a dry-run listing the services and components that will be provisioned on specified provider's 
-  				platform.
-  * <code>terraform validate</code> parses through the terraform code to ensure that they are both syntactically correct and runtime 
-  				    error free.
-  * <code>terraform apply</code> this command provision's the described services on the providers platform.
+  * <code>terraform init</code> 
+  * <code>terraform plan</code>	
+  * <code>terraform validate</code> 
+  * <code>terraform apply</code> 
+  <details> - terraform init initialises and downloads the provider's binary as described in the .tf files
+ - terraform plan is a dry-run listing the services and components that will be provisioned on specified provider's platform.
+- terraform validate parses through the terraform code to ensure that they are both syntactically correct and runtime error free.
```

Comment:
Although, seems that `terraform validate` that exist in this commit was lost in recent change :P